js/global.js

```diff
@@ -0,0 +1,99 @@
+const input = document.querySelector("input");
+const button = document.querySelector("button");
+const magnify = document.querySelector(".fa-magnifying-glass");
+const loadingSpinner = document.querySelector(".fa-spinner");
+const searchList = document.querySelector(".search-results");
+const marqueeElement = document.querySelector(".marquee");
```

Comment:
is this relevant to the company page?
the global file should contain only variables that are relevant to all of the files